view/player/tmpl/default.html.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright	Copyright (C) 2011 - 2014 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license		GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link		http://github.com/nooku/nooku-files for the canonical source repository
+ */
+defined('KOOWA') or die;
+?>
+
+<ktml:style src="assets://files/css/plyr.css" />
+<ktml:script src="assets://files/js/plyr/plyr.js" />
+<script>
+    jQuery(function($){
```

Comment:
@rmdstudio Few things: 
* You should call `<?= helper('behavior.jquery') ?> on this file as it uses jQuery. 
* You should use `kQuery` instead of `jQuery` as the variable name. This makes sure that we only use on jQuery instance. (Joomla templates are known to include multiple)
* plyr.js should also use kQuery instead of jQuery. 

Let me know if you have any questions about the usage of kQuery. You can find examples on `KTemplateHelperBehavior`